classes/core/PKPApplication.inc.php

```diff
@@ -61,6 +61,13 @@
 define('WORKFLOW_TYPE_EDITORIAL', 'editorial');
 define('WORKFLOW_TYPE_AUTHOR', 'author');
 
+// Constant used to distinguish whether metadata is enabled and whether it
+// should be requested or required during submission
+define('METADATA_DISABLE', 0);
```

Comment:
I'm a little ambivalent about putting these here. Maybe in `DataObject`, or `Form`, depending on whether you feel they're better aligned with the model or the controller?

Reply:
:+1: I moved them into `Context`.